ads/smartadserver.md

```diff
@@ -16,14 +16,29 @@ limitations under the License.
 
 # SmartAdServer
 
+## Supported Smart AdServer parameters in the amp-ad tag
+
+All of the parameters listed here should be prefixed with "data-" when used.
+
+| Parameter name | Description                         | Required |
+|----------------|-------------------------------------|----------|
+| site           | Your Smart AdServer Site ID         | Yes      |
+| page           | Your Smart AdServer Page ID         | Yes      |
+| format         | Your Smart AdServer Format ID       | Yes      |
+| domain         | Your Smart AdServer call domain     | Yes      |
+| target         | Your targeting string               | No       |
+| tag            | An ID for the tag containing the ad | No       |
+
+Note: If any of the required parameters is missing, the integration won't be able to work correctly,
+and a "noContentAvailable" notification will be returned by the Smart AdServer integration.
```

Comment:
Cool. Could you explain in what form the notification is?

Meantime, we recently added new API: [renderStart](https://github.com/ampproject/amphtml/blob/master/ads/README.md#methods-available-to-the-ad)
Could you take a look and implement that API along with `noContentAvailable`?
It helps AMP to provide a better ad loading experience.